docs/google_provider_setup.md

```diff
@@ -127,8 +127,9 @@ variables together for any one provider. This can be changed to an identifier th
 sense for your individual use case.
 If following the quickstart, make sure this identifier matches what you used there.
 
-- **`PROVIDER_GOOGLEQUICKSTART_GOOGLE_IMPERSONATE`**: An administrative email address on your organization's
+- **`PROVIDER_GOOGLEQUICKSTART_GOOGLE_IMPERSONATE`**: (string) An administrative email address on your organization's
 domain, the identity of which can be assumed by `sso`.
-- **`PROVIDER_GOOGLEQUICKSTART_GOOGLE_CREDENTIALS`**: The path to the JSON file downloaded at the time of
+- **`PROVIDER_GOOGLEQUICKSTART_GOOGLE_CREDENTIALS`**: (string) The path to the JSON file downloaded at the time of
 service account creation above. There is no reason why this file should ever be accessed by any
 person or service other than `sso`; ensure that file permissions are set accordingly.
+- **`PROVIDER_GOOGLEQUICKSTART_GROUPCACHE_INTERVAL_REFRESH`**: (time.Duration - default: `10m`) The interval the group cache should refresh at. #TODO:set default/test?
```

Comment:
is this TODO now effectively done with https://github.com/buzzfeed/sso/pull/246?

Reply:
It is! Good catch - thank you!